classes/oc/captcha.php

```diff
@@ -84,7 +84,7 @@ public static function check($name = '', $ajax = FALSE)
         //d(Session::instance()->get('captcha_'.$name));
         //d(Session::instance()->get('captcha_'.$name) == strtolower(core::post('captcha')));
 
-        if (core::config('advertisement.captcha') == FALSE) // Captchas are disabled
+        if (core::config('advertisement.captcha') == FALSE AND core::config('general.captcha') == FALSE) // Captchas are disabled
```

Comment:
The code I suggested will work on Common, since I check first if its not null which is the default return in case the config doesnt exists ;)